src/bot_creds.sh

```diff
@@ -0,0 +1,2 @@
+BOT_TOKEN="0"
```

Comment:
```suggestion
BOT_TOKEN="ADD_YOUR_TELEGRAM_BOT_TOKEN"
```

I would also like to suggest changing the file name to `bot_credentials.sh` to clarify its meaning.

Reply:
Updated in latest version of PR